AutoRest/Generators/Extensions/Extensions.Tests/MappingTests.cs

```diff
@@ -33,10 +33,9 @@ public void TestInputMapping()
             generator.Generate(clientModel).GetAwaiter().GetResult();
             string body = settings.FileSystem.ReadFileAsText("X:\\Payload.cs");
```

Comment:
I think this will fail on non-windows ... (which is maybe why this was set to skip?)